skyportal/handlers/api/instrument.py

```diff
@@ -301,6 +302,17 @@ def add_tiles(instrument_id, instrument_name, regions, field_data):
                 },
             }
 
+            geometry = []
+            for coord in coords:
+                tab = []
+                for i in range(len(coord.ra.deg)):
+                    tab.append([coord.ra.deg[i], coord.dec.deg[i]])
+                tab.append([coord.ra.deg[0], coord.dec.deg[0]])
+                geometry.append(tab)
+
+            contour['type'] = "Feature"
+            contour['geometry'] = {'coordinates': geometry, 'type': 'MultiLineString'}
+            contour = {'features': [contour], 'type': "FeatureCollection"}
```

Comment:
This is a bit of a confusing statement.  Perhaps rewrite in its entirety as:

```python
contour = {
    'type': 'FeatureCollection',
    'features': [
        {
            'type': 'Feature',
            'geometry': {
                'type': 'MultiLineString'
                'coordinates': geometry,
            },
        },
    ]
}
```

If I unpacked that correctly!  But you can see that it's quite subtle to unpack it in your head, but once unpacked it's quite easy to follow.

Reply:
yes much better.